entrypoint.sh

```diff
@@ -13,3 +13,8 @@ wget -N https://s3.amazonaws.com/Minecraft.Download/versions/${MINECRAFT_VERSION
 # Run the world renders (One pass to make map, one to generate points of interests)
 overviewer.py --config /home/minecraft/config.py
 overviewer.py --config /home/minecraft/config.py --genpoi
+
+# Add GMaps API key if API_KEY environment variable is set
+if [ "$API_KEY" ]; then
```

Comment:
Can you match the pattern on line 5?
```bash
if [ -z "$API_KEY" ]; then
```

Reply:
Is the control structure ok? I would need to add a negation since I only want to run the command when the environment variable is set. Like so?

```bash
if [ ! -z "$API_KEY" ]; then
```

Reply:
Oh geez, You're right. I'm sorry. Ignore this whole thread.